### ISSUE TITLE
Non-ASCII characters not being filled in form

### ISSUE THREAD

**Issue by lagavuliner** [johntayl/pdffiller#95]
Hi, thank you for your script.
I can't get the non-ASCII characters into the completed PDF, do you know how I can troubleshoot and/or fix this problem?

> var pdfFiller   = require('pdffiller');
> var sourcePDF = "test/test.pdf";
> var destinationPDF =  "test/test_complete.pdf";
> var data = {
>     "firstName" : "String with some special characters čćšđž"
> };
>  
> pdfFiller.fillForm( sourcePDF, destinationPDF, data, function(err) {
>     if (err) throw err;
>     console.log("In callback (we're done).");
> });

**Comment by terrisgit** [comment]
I'm going to try https://github.com/eemeli/pdf-form-fill

**Comment by michaeldoubek** [comment]
Same issue here for Czech characters

**Comment by rbadr** [comment]
Hello @snowlep @mikeoaklet 👋 
Did you find a solution to your problem ? I'm encountering the same issue 😅

**Comment by lagavuliner** [comment]
@rbadr No, and I scrapped my project for other reasons so I didn't find a work-around.
